frontend/app/components/redesign/components/landing/PillTag.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export type PillTagProps = {
+  children: React.ReactNode
+  variant?: 'default' | 'active'
+  className?: string
+}
+
+export const PillTag = ({
+  children,
+  variant = 'default',
+  className = ''
+}: PillTagProps) => {
+  const textColorClass =
+    variant === 'active' ? 'text-text-secondary' : 'text-text-helper'
+
+  return (
+    <div
```

Comment:
Use `<span>` instead of div. Even better, these are meant as list of tags (`<ul><li>`). So, maybe keep `<span>` here and on parent, use `<ul><li>` while keeping same style.